Portage/EBuild.hs

```diff
@@ -125,6 +125,9 @@ showEBuild now ebuild =
      Nothing -> id
      Just pn -> ss "MY_PN=". quote pn. nl.
                 ss "MY_P=". quote "${MY_PN}-${PV}". nl. nl).
+  (case my_pn ebuild of
```

Comment:
I think you can move `ss "S=". quote ("${WORKDIR}/${MY_P}").` right under case one line above.

Reply:
Yes you're right! I will fix this

Reply:
Ok this is done now.